torchbenchmark/models/dlrm/__init__.py

```diff
@@ -51,8 +51,9 @@ def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
         arch_mlp_bot = "512-512-64"
         arch_mlp_top = "1024-1024-1024-1"
         data_generation = "random"
-        mini_batch_size = 2048
-        num_batches = self.batch_size
+        mini_batch_size = self.DEFAULT_TRAIN_BSIZE if batch_size is None else batch_size
```

Comment:
Can we just set `mini_batch_size=self.batch_size`? Because in `super().__init__()` we will set the default batch size according to the value of `test` is `train` or `eval`.

Reply:
that's strange, I manually added line 55 `self.batch_size = mini_batch_size`, because run.py was throwing an error that batch_size is not set.

Reply:
Can I have the stack trace? run.py shouldn't throw error when batch_size is `None`...

Reply:
```
(py38) aaronshi@a100-st-p4d24xlarge-18:/scratch/aaronshi/work/benchmark$ python run.py dlrm -d cuda -m eager --profile -t train --bs 4096
Traceback (most recent call last):
  File "run.py", line 228, in <module>
    m = Model(device=args.device, test=args.test, jit=(args.mode == "jit"), batch_size=args.bs, extra_args=extra_args)
  File "/scratch/aaronshi/work/benchmark/torchbenchmark/util/model.py", line 16, in __call__
    obj = type.__call__(cls, *args, **kwargs)
  File "/scratch/aaronshi/work/benchmark/torchbenchmark/models/dlrm/__init__.py", line 54, in __init__
    mini_batch_size = self.batch_size
AttributeError: 'Model' object has no attribute 'batch_size'
```

Reply:
This issue is fixed after cherry-picking: https://github.com/pytorch/benchmark/pull/1094